deployment/ccip/changeset/cs_chain_contracts.go

```diff
@@ -351,6 +353,283 @@ func UpdateOnRampsDestsChangeset(e deployment.Environment, cfg UpdateOnRampDests
 	}}, nil
 }
 
+type UpdateOnRampDynamicConfig struct {
+	// UpdatesByChain is a mapping of source -> update.
+	UpdatesByChain map[uint64]OnRampDynamicConfigUpdate
+	// Disallow mixing MCMS/non-MCMS per chain for simplicity.
+	// (can still be achieved by calling this function multiple times)
+	MCMS *MCMSConfig
+}
+
+type OnRampDynamicConfigUpdate struct {
+	FeeQuoter          common.Address
```

Comment:
let's fetch feequoter address from state. Whenever possible we use chainselector to load corresponding contract addresses from state instead of taking it as user input. It reduces the chance of error.